packages/consumption/test/modules/requests/RequestsIntegrationTest.ts

```diff
@@ -318,6 +318,17 @@ export class RequestsGiven {
 
         await this.moveOutgoingRequestToStatus(this.context.givenLocalRequest, params.status);
 
+        if (waitForExpiration && params.content.expiresAt) {
+            const dateString = params.content.expiresAt.dateTime.toISO();
+            if (dateString) {
+                const date = CoreDate.from(dateString);
+                while (CoreDate.utc().isBefore(date)) {
+                    await sleep(1000);
+                }
+                await sleep(1000);
+            }
+        }
+
```

Comment:
In my opinion it is much easier to remove this here and simply add a sleep to those tests that require an expiration time to pass. Looking only at the tests, I didn't understand why we don't have a sleep there, since the true that was passed to `anOutgoingRequestWith` doesn't make it obvious, what's going on. Also, you don't have to adjust `ICoreDate` then. Exposing a DateTime there doesn't feel like something we want to do.